addons/skin.estuary/xml/DialogAddonInfo.xml

```diff
@@ -217,7 +217,7 @@
 								</item>
 								<item>
 									<label>$LOCALIZE[126]:</label>
-									<label2>$INFO[ListItem.Property(Addon.Status)]</label2>
+									<label2>$VAR[AddonStatus]</label2>
```

Comment:
you can accomplish this without a `$VAR[]` if you use the `$INFO[infolabel,prefix,postfix]` format.
this format will hide the prefix & postfix if the infolabel returns empty.

```
$INFO[ListItem.Property(Addon.Status)]$INFO[ListItem.Property(Addon.UpdateTargetVersion),[CR]($LOCALIZE[19114]: ,)]
```

Reply:
done + the requested doc